geopandas/base.py

```diff
@@ -1,9 +1,11 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you remove this? (not that I feel strongly against this or so, just for consistency)

Reply:
I added this because it wasn't working for one of my environments. I think it was python 2.7. I would recommend keeping it and adding it to the other files as well.

Reply:
I suppose it was because you had a non-ascii character below (for which I did an inline suggestion).

Reply:
Looks like it won't be necessary anymore with the drop of python 2 support.